nxdrive/direct_edit.py

```diff
@@ -63,6 +63,7 @@ def _is_lock_file(name: str) -> bool:
 class DirectEdit(Worker):
     localScanFinished = pyqtSignal()
     directEditUploadCompleted = pyqtSignal(str)
+    directEditUploadFailed = pyqtSignal(str)
```

Comment:
I would prefer to reuse `directEditError` as we are doing at several places. WDYT?